Assets/_BForBoss/_Core/Scripts/Character/PlayerDashBehaviour.cs

```diff
@@ -8,7 +8,7 @@ namespace BForBoss
 {
     public interface IPlayerDashEvents
     {
-        void OnDashStarted();
+        void OnDashStopped();
```

Comment:
So I realized that `OnDash` get's executed few timers per frame. 

Which means accrue would get called more than once. 
However, OnDashStopped is guarenteed to be called once. 

So I set the accruement `onDashStopped` instead.